src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -117,16 +118,22 @@ def get_module_TLC_container(self, table):
 """
 Crop config as per yang models,
 This Function crops from config only those TABLEs, for which yang models is
-provided.
+provided. The Tables without YANG models are stored in
+self.tablesWithOutYangModels.
 """
-def cropConfigDB(self, croppedFile=None, allowExtraTables=True):
+def cropConfigDB(self, croppedFile=None):
 
     for table in self.jIn.keys():
         if table not in self.confDbYangMap:
-            if allowExtraTables:
-                del self.jIn[table]
-            else:
-                raise(Exception("No Yang Model Exist for {}".format(table)))
+            # store in tablesWithOutYang
+            self.tablesWithOutYang[table] = self.jIn[table]
+            del self.jIn[table]
+
+    if len(self.tablesWithOutYang):
+        print("Note: Below table(s) have no YANG models:")
+        for table in self.tablesWithOutYang.keys():
+            print(unicode(table), end=", ")
+        print()
```

Comment:
what is this print() for?

Reply:
print(unicode(table), end=", ") prints all extra tables in one line, then print() is just used to change line.

Reply:
Same as https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-mgmt/sonic_yang_ext.py